app/controllers/people_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @task = Task.all
```

Comment:
Per our naming conventions, we generally want to use the plural when he variable is storing all of the items.